apps/website/src/app/problems/[problemSlug]/_components/problem/body.tsx

```
@@ -0,0 +1,12 @@
+import { getProblemBody } from "@easyshell/problems"
+import { MDXRemote } from "next-mdx-remote-client/rsc"
+
+import { customComponents } from "@/mdx-components"
+
+export async function ProblemBody({ slug }: { slug: string }) {
+  const text = await getProblemBody(slug)
+  console.log(text)
+  // return <div>{text}</div>
+  return <MDXRemote source={text} components={customComponents} />
+  return <div>bkjfdb</div>
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)